frontend/config/routes.rb

```diff
@@ -19,8 +19,8 @@
   end
 
   get '/cart', to: 'orders#edit', as: :cart
-  patch '/cart', to: 'orders#update', as: :update_cart
-  put '/cart/empty', to: 'orders#empty', as: :empty_cart
+  match '/cart', to: 'orders#update', as: :update_cart, via: [:put, :patch]
+  match '/cart/empty', to: 'orders#empty', as: :empty_cart, via: [:put, :patch]
```

Comment:
And here too ? And you mentioned fix routes. Not sure what is the problem this PR trying to fix ?